packages/@okta/vuepress-site/guides/sign-into-web-app/define-callback/aspnetcore/define-route.md

```diff
@@ -0,0 +1,3 @@
+The Okta ASP.NET Core SDK will configure and host this route for you. By default, the route is hosted at `/authorization-code/callback`.
```

Comment:
The Okta ASP.NET Core SDK configures and hosts this route for you. By default, the route is hosted at `/authorization-code/callback`.